contrib/kind.sh

```diff
@@ -234,6 +234,34 @@ kubectl get secrets -o jsonpath='{.items[].data.token}' > /tmp/kind/token
 pushd ../go-controller
 make
 popd
+
+if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then
```

Comment:
https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

> GITHUB_ACTIONS	Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.